pandas/tests/series/methods/test_convert_dtypes.py

```diff
@@ -225,3 +225,10 @@ def test_convert_bool_dtype(self):
         # GH32287
         df = pd.DataFrame({"A": pd.array([True])})
         tm.assert_frame_equal(df, df.convert_dtypes())
+
+
+class TestDataFrameConvertDtypes:
+    def test_convert_empty(self):
+        # Empty DataFrame can pass convert_dtypes, see GH#40393
+        df = pd.DataFrame().convert_dtypes()
+        assert df.empty
```

Comment:
There is a similar file in the `pandas/tests/frame/methods` directory, which is probably the more suitable place to put this test

Reply:
Thanks - yeah I was a bit uncertain where best to leave that.

Reply:
Oops I see - I totally didn't notice there was separate folder hierarchies for series and frame tests. Thanks for correcting me.